medor_custom/templates/trial_subscription_form.xml

```diff
@@ -0,0 +1,125 @@
+                                        <div name="email_confirmation_container" t-attf-class="form-group #{error and 'email_confirmation' in error and 'has-error' or ''}">
+                                            <label class="col-md-3 col-sm-4 control-label" for="email_confirmation">Confirm Email</label>
+                                            <div class="col-md-7 col-sm-8">
+                                                <input type="email" class="form-control" name="email_confirmation" required="True" t-attf-value="#{email_confirmation or ''}" />
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="bottom-line" style="margin-left:178px;margin-top:0px;width:61%" />
+                                    <br />
+                                    <table style="margin-left:195px">
+                                        <tr>
+                                            <td width="80%">
+                                                <div class="g-recaptcha" t-att-data-sitekey="website.recaptcha_site_key" data-theme="green" />
+                                                <br />
+                                            </td>
+                                            <td>
+                                                <div class="form-group">
+                                                    <div class="col-md-offset-3 col-sm-offset-4 col-sm-8 col-md-7">
+                                                        <button class="btn btn-primary btn-lg">Send</button>
+                                                    </div>
+                                                </div>
+                                            </td>
+                                        </tr>
+                                    </table>
+                                    <br />
+                                </form>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="oe_structure" />
+                </div>
+            </t>
+        </template>
+
+        <template id="template_trial_subscription_thanks" name="Trial Subscription Thanks" page="True">
+            <t t-call="website.layout">
+                <div id="wrap">
+                    <div class="oe_structure" />
+                    <div class="container">
+                        <h1>Thank you!</h1>
+                        <div class="row">
+                            <div class="col-md-8">
+                                <div class="alert alert-success">
+                                    Your subscription has been successfully registered.
+                                    Check your emails for instructions to set your password.
+                                    <button type="button" class="close" data-dismiss="alert">&amp;times;</button>
```

Comment:
we can remove this ! it's useless... I know it comes from my shit

Reply:
The close button you mean?

Reply:
yes